docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -44,7 +59,7 @@ PUT _transform/ml_hostriskscore_pivot_transform
 GET ml_host_risk_score/_search (or _count)
 ```
 
-#### 6. Create the `ml_host_risk_score_latest` index with appropriate mappings
+#### 7. Create the `ml_host_risk_score_latest` index with appropriate mappings
```

Comment:
Should we also specify here, that index should be space aware?
like `ml_host_risk_score_latest_default` or `ml_host_risk_score_latest_<space>`

Reply:
Ah. That's a good point. This is not something that gets taken care of by generate_scripts.py so yes, I'll add that in here.